### ISSUE TITLE
Artifacts are missing on latest release

### ISSUE THREAD

**Issue by ruudk** [jolicode/castor#568]
Castor tells me that there is a new version without telling me how to upgrade.

I took the previous upgrade script and ran it:

```
curl "https://castor.jolicode.com/install" | bash -s -- --install-dir /Users/Ruud/.local/bin
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  6502    0  6502    0     0    97k      0 --:--:-- --:--:-- --:--:--   99k
🦫  Castor installer

Environment check
  [*] cURL is installed
  [*] PHP is installed
  [*] PHP version: 8.3.13
  [*] Your architecture (arm64) is supported

Download
  Downloading https://github.com/jolicode/castor/releases/latest/download/castor.darwin-arm64.phar...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     9    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (56) The requested URL returned error: 404
```

**Comment by ruudk** [comment]
Oke, please be aware that Castor prompts this to everyone:

> A new Castor version is available (v0.20.0, currently running v0.19.1).

**Comment by ruudk** [comment]
Or is that gone now, since it's marked as `pre-release`.

**Comment by pyrech** [comment]
I have manually uploaded artifacts to the release and re-marked it as latest. Everything should be good now.

We will need to fix #520 as soon as possible.

Thanks for the issue
